taworks/taform/views.py

```diff
@@ -42,7 +156,8 @@ def apply(request):
     if request.method == 'POST':
         num = [x for x in models.Course.objects.all()]
         s_form = models.StudentForm(request.POST, request.FILES or None)
-        a_forms = [models.ApplicationForm(request.POST, prefix=str(x), instance=models.Application()) for x in range(len(num))]
+        a_forms = [models.ApplicationForm(request.POST, prefix=str(x), 
```

Comment:
just making length of line smaller?

Reply:
Yep, try to keep it 80 lines whenever possible.